AutoGrade/views.py

```diff
@@ -451,7 +451,7 @@ def assignment_aggregate_report(request, assignment_id):
     submissions = assignment.get_student_and_latest_submissions()
```

Comment:
This will also effect `admin/assignment_aggregate_report.html`

Reply:
Don't think it would since we're basically taking care of the third element in the tuple over here.